src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,78 @@
-import React from 'react';
+import React, { memo, useCallback } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  onSelect: (todoId: number) => void;
+  selectedTodo: Todo | null;
+};
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-      <tr data-cy="todo" className="has-background-info-light">
-        <td className="is-vcentered">2</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">quis ut nam facilis et officia qui</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye-slash" />
-            </span>
-          </button>
-        </td>
-      </tr>
+export const TodoList: React.FC<Props> = memo(({
+  todos,
+  onSelect,
+  selectedTodo,
+}) => {
+  const iconClassName = useCallback((currentTodo: Todo): string => {
+    return currentTodo === selectedTodo ? 'far fa-eye-slash' : 'far fa-eye';
+  }, [selectedTodo]);
```

Comment:
Use classnames library for this.

Reply:
How should I implement it? Doesn't classNames libray only add or remove classes not replace them?

Reply:
Just use correct conditions.
```
className={classnames('class1', {
  class2: condition2,
  class3: condition3,
})}
```